plugins/org.jkiss.dbeaver.ext.ui.tipoftheday/tips.xml

```diff
@@ -111,6 +111,13 @@
 		 table by selecting the Replace method in the data load settings. Supported by Oracle, MySQL, PostgreSQL, Firebird, and SQLite.</tip>
 	<tip>You can improve data import performance by enabling <b>Use multi-row value</b> insert in the data transfer settings.
 		 Additionally, you can increase the rows count in the <b>multi-insert batch</b>.</tip>
+	<tip>You can jump to a column in the result set using <commandRef commandId="org.jkiss.dbeaver.core.resultset.grid.gotoColumn"/> command.</tip>
+	<tip>To view script execution results sequentially in one tab, enable the option from the SQL Editor context menu: <b>Execute -> Toggle execution results orientation</b>.</tip>
```

Comment:
```suggestion
	<tip>To view result tabs vertically rather than horizontally, right-click anywhere in the SQL editor and enable <b>Execute -> Toggle execution results orientation</b>.</tip>
```